mainmatter/introduction/pipeline/index.tex

```diff
@@ -0,0 +1,96 @@
+\section{The Graphics Pipeline} \label{sec:pipeline}
+\begin{sectionmeta}
+To understand the following sections it is recommended to gather an understanding of what the graphics pipeline is.
+The following section is based on a mix of the official documentation for the graphics pipelines for Direct3D and OpenGL \cite{khronos????pipeline, microsoft????pipeline}.
+\end{sectionmeta}
+
+The graphics pipeline can be compared to the \gls{CPU} pipeline in the way that it is a number of stages that each command goes through.
```

Comment:
for meget "that" i den sætning.

Reply:
<crime_rider> removes two </crime_rider>